src/style.css

```diff
@@ -17,12 +19,84 @@ body {
 
 .heading {
   color: var(--heading-color);
+  font-size: 3rem;
```

Comment:
- [optional] Great job following the wireframe design and making your style unique. Please, for user-friendliness, kindly make this circled colors more visible. I am sure you won't love if someone stress his/her eyes to read something caused by your design. 😄 nice work. 
![Screenshot (1315)_LI](https://github.com/Titus-Kiplagat/leaderboard/assets/65298048/b4e94525-4f27-49b8-868d-1cebc6456031)

Reply:
Yeah, that's absolutely true. Ensuring good text contrast is a fundamental aspect of web accessibility. Making sure that circled colors are more visible not only improves the overall user experience but also ensures that the design is inclusive and can be easily accessed by everyone, including those with visual impairments. Thanks for the feedback, and I'm glad you appreciate the effort to enhance user-friendliness! 😄

Reply:
Thank you for your positive feedback! I'm glad to hear you're happy with the project. If you think it's worth it, I would greatly appreciate a star for my effort. 🌟